### ISSUE TITLE
Model#find_by_hashid method is falling back on the standard find method

### ISSUE THREAD

**Issue by frederickdubois** [jcypret/hashid-rails#35]
Hi!

Currently using version 1.0.0 and just noticed that the Model#find_by_hashid method returns my models even if I pass the original integer id values...

If I open the rails console and type `Company.find_by_hashid(1)` I get my company with id=1... same thing with other ids... Normally, passing 1 as the argument here should not return the company...

**Comment by frederickdubois** [comment]
As a workaround, I added this validation for now to make sure in a specific sensitive page no visitor can guess the ids with incremental integer values...

```
# if I enter "1" as the id in the url, the find_by_hashid still works...
@company = Company.find_by_hashid!(params[:id])
throw 'Used incorrect ID to access offer' if @company.hashid != params[:id]
```

**Comment by jcypret** [comment]
Thanks @frederickdubois for opening this issue. I have a PR in progress to correct the behavior: https://github.com/jcypret/hashid-rails/pull/41. Should be ready to go soon.

**Comment by jcypret** [comment]
Issues resolved and release in version 1.2.0.
